src/translations/fr/legal-notice.mdx

```diff
@@ -1,18 +1,41 @@
-# Mentions légales
+# Mentions légales de Open Terms Archive
```

Comment:
```suggestion
# Mentions légales
```

or 
```suggestion
# Mentions légales d'Open Terms Archive
```

Reply:
Used `Mentions légales d'Open Terms Archive` to be in line with https://github.com/betagouv/juridiques/blob/main/Mentions-l%C3%A9gales.md